lib/core.js

```diff
@@ -431,15 +439,6 @@ declare class Promise<R> {
     static reject<T>(error?: any): Promise<T>;
     static all<T, Elem: Promise<T> | T>(promises: Array<Elem>): Promise<Array<T>>;
     static race<T, Elem: Promise<T> | T>(promises: Array<Elem>): Promise<T>;
-
-    // Non-standard APIs common in some libraries
```

Comment:
Hmm... So these are indeed non-standard, but they're here because they're commonly understood in the places they do exist (and thus there's not a great reason for Flow to *not* understand them).

It's one thing for libraries or polyfills to ship these and claim standard-conformance, but it's another for Flow to have a libdef so it understands what'll happen if someone does happen to use them.